Columns w/ Icons

ryanmparrish opened the pull request:
- accordion header icon alignment
 - adjusted column icon-elem styles and logic to display icon margin in p and header tags
 - added option to section-metadata: grid | span-auto - (see the customers `V2` page w/ a single quick-link button/title)
 - fixed `btn.blue` brand colors

Fix 
- Icon wrapping for `<p>` - [357](https://github.com/aemsites/creditacceptance/issues/357)
- UL accessiblity - [#366](https://github.com/aemsites/creditacceptance/issues/366)
- Big-icons - [373](https://github.com/aemsites/creditacceptance/issues/373)

Customers pg: (column-icons)
- Before: https://main--creditacceptance--aemsites.aem.page/customers
- After: https://rparrish-column-icons--creditacceptance--aemsites.aem.page/customers

Contact Us pg: (grid-auto, column-icons) - I added a V2 page w/ updated authoring. This is the preferred page
- Before: https://main--creditacceptance--aemsites.aem.page/about/contact-us
-- Before: https://main--creditacceptance--aemsites.aem.page/about/contact-us-v2
- After: https://rparrish-column-icons--creditacceptance--aemsites.aem.page/about/contact-us
-- After: https://rparrish-column-icons--creditacceptance--aemsites.aem.page/about/contact-us-v2

Network pg: (big-icons)
- Before: https://main--creditacceptance--aemsites.aem.page/dealers/join-our-network
- After: https://rparrish-column-icons--creditacceptance--aemsites.aem.page/dealers/join-our-network

Testing criteria - Check links against the live site and make sure all looks the same. (for the most part 😉)
Live site
https://www.creditacceptance.com/customers
https://main--creditacceptance--aemsites.aem.page/about/contact-us
https://main--creditacceptance--aemsites.aem.page/dealers/join-our-network

ryanmparrish commented:
@kailasnadh790 - FYI, [The customers page](https://rparrish-column-icons--creditacceptance--aemsites.aem.page/customers) has a known issue. @jordanpetersen27  - In the interest of keeping the content fluid on all viewports I think we should allow these app buttons to live below the image/copy of the columns. They have a -100 top position that isn't really ideal when supporting differing viewport sizes. LMK if that's ok and I can remove those conflicting styles

<img width="982" alt="Screenshot 2025-02-28 at 3 43 43 PM" src="https://github.com/user-attachments/assets/33ea9bf2-ad4e-47eb-af6b-ba2ad95697aa" />

kailasnadh790 commented on libs/utils/decorate.js:
```diff
@@ -144,7 +145,15 @@ export function decorateGridSection(section, meta) {
       }
+      // if single span-auto row, add span-auto class to all rows
+      if (gridValues[0] === 'span-auto' && gridValues.length === 1) {
+        row.classList.add('span-auto');
```
@ryanmparrish do we have to do the same for decorateGridSectionGroups ?

ryanmparrish replied:
Good call. I have added this feature to the `decorateGridSectionGroups()` function as well

kailasnadh790 commented:
Looks great!! Just the above question.

kailasnadh790 commented:
@ryanmparrish check the button colours on the home page. some regression ?
![image](https://github.com/user-attachments/assets/d0043fc3-652d-456e-ab15-4de2463bfaab)

ryanmparrish commented:
@kailasnadh790  - Thanks for the feedback. I have removed the button color change regression and removed that `top: -100` style. LMK if you see any other snags.

tbxsaugat reviewed (approved):
Great.. LGTM
